protected/migrations/m170821_053800_cera_clean.php

```diff
@@ -3,58 +3,59 @@
 class m170821_053800_cera_clean extends OEMigration
 {
 
-	// Use safeUp/safeDown to do migration with transaction
-	public function safeUp()
-	{
-	    $source = new ImportSource;
-	    $source->name = 'Fivium Australia';
+    // Use safeUp/safeDown to do migration with transaction
+    public function safeUp()
+    {
+        $source = new ImportSource;
+        $source->name = 'Fivium Australia';
         if (!$source->save()) {
-            throw new CDbException("Unable to save source $source->name");
+            throw new CDbException("Unable to save source: {$source->getErrors()}");
```

Comment:
getErrors returns and array so this won't work. You'll need to implode it perhaps.